mlflow/entities/span_event.py

```diff
@@ -47,14 +47,15 @@ def from_exception(cls, exception: Exception):
             },
         )
 
-    def _get_stacktrace(self, error: BaseException) -> str:
-        """Get the stacktrace of the parent error."""
-        msg = repr(error)
-        try:
-            if sys.version_info < (3, 10):
-                tb = traceback.format_exception(error.__class__, error, error.__traceback__)
-            else:
-                tb = traceback.format_exception(error)
-            return (msg + "\n\n".join(tb)).strip()
-        except Exception:
-            return msg
+
+def _get_stacktrace(error: BaseException) -> str:
```

Comment:
couldn't this also just be a static method of the class? The error before was because it was declared as an instance method right?

Reply:
Yeah that works too. I though it's kinda better to define outside as it does nothing with span event, but this method will likely only used within this class so putting it as static method makes more sense now.